src/utilities/browser.ts

```diff
@@ -0,0 +1,32 @@
+export const IS_MAYBE_MAC = typeof navigator !== 'undefined' &&
```

Comment:
issue: the utils in this file need to first check they are in a browser environment so that they don't cause issues in SSR

Reply:
I started reviewing textarea PR on mobile but on second look I don’t think SSR is an issue since you first check that the properties are not undefined; we can verify before merge 👍

Reply:
Yes I'm expecting the checks here to be enough.